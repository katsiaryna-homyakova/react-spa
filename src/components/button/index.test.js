import React from 'react';
import Button from './index';
import { shallow } from 'enzyme';;


describe("button component", () => {
  it('renders correctly', () => {
    const component = shallow(<Button text="Some button" classes="some classes" handleClick="someEvent" />);
    expect(component).toMatchSnapshot();
  });
});