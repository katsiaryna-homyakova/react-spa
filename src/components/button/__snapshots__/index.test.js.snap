// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button component renders correctly 1`] = `ShallowWrapper {}`;
